services/template/README.md

```diff
@@ -22,22 +22,23 @@ npm install # install required Node.js dependencies
 
 ## Configuration
 
-To configure the different microservices, there are a number of environment variables, the user can define in order to customize the configurations. The template service does not use any of them by default.
+To configure the different microservices, there are several environment variables, the user can define to customize the configurations.
+The template service does not use any of them by default.
 
 ## Management
 
 ### Start
 
 ```bash
-cd lisk-service/services/template # move into root directory of the template microservice
+cd lisk-service/services/template # move into the root directory of the template microservice
 npm start # start the microservice
 ```
 
 Use the `framework/bin/moleculer_client.js` and `framework/bin/moleculer_subscribe.js` clients to test specific service endpoints.
 
 Once the process is verified as running correctly, `CTRL+C` and start the process with `PM2`. This will fork the process into the background and automatically recover the process if it fails.
```

Comment:
```suggestion
Once the process is verified as running correctly, press `CTRL+C` and start the process with `PM2`. This will fork the process into the background and automatically recover the process if it fails.
```